platform/ios/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Other changes
 
 * Xcode 8.0 or higher is now recommended for using this SDK. ([#8775](https://github.com/mapbox/mapbox-gl-native/pull/8775))
+* Fixed a crash when calling `MGLMultiPolygon.coordinate` [#8713](https://github.com/mapbox/mapbox-gl-native/pull/8713)
```

Comment:
Also note (perhaps in the “Annotations” section) that the `coordinate` property on MGLPolyline and MGLPolygon now returns a sensible value.